components/pipefy/actions/create-card/create-card.mjs

```diff
@@ -35,17 +74,34 @@ export default {
   }
   */
 
-    if (!this.graphql_query) {
-      throw new Error("Must provide graphql_query parameter.");
+    let mutationString = `
```

Comment:
Is there a better way to create a query with variables?
Maybe like in **Shopify Partner** or **Shopify**?